spec/venue_spec.rb

```diff
@@ -44,12 +40,59 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
+
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  describe '#over_capacity?' do
+    it 'is false' do
```

Comment:
Nice job testing for both `true and false`. It is perfectly fine to write separate tests or you can combine them into a single test.